assets/capability/capabilities/speaker_playing.json

```diff
@@ -16,6 +16,36 @@
   "uiComponent": "media",
   "uiQuickAction": true,
   "$flow": {
+    "triggers": [
+      {
+        "id": "speaker_playing_true",
```

Comment:
Will the triggers be automatically generated from this?

Reply:
Yes

Reply:
But it might not happen because the capability doesn't start with `alarm_`.

Reply:
it worked in my testing, I think all boolean capabilities automatically try to add flow cards with `_true` and `_false`